.github/workflows/publish.yml

```diff
@@ -10,16 +10,16 @@ permissions:
 
 jobs:
   publish_maplibre_gl_platform_interface:
-    uses: ./.github/workflows/publish_single.yml
+    uses: maplibre/flutter-maplibre-gl/.github/workflows/publish-single.yml@main
```

Comment:
I think this change is not needed, only the dash in the filename was wrong :)

Reply:
I tested it but the ci failed

- the @main is needed, see https://github.com/josxha/flutter-maplibre-gl/actions/runs/9181129759/workflow
- and the owner/repo seems to be required too, see https://github.com/josxha/flutter-maplibre-gl/actions/runs/9181145594

Reply:
> [Invalid workflow file: .github/workflows/publish.yml#L13](https://github.com/josxha/flutter-maplibre-gl/actions/runs/9181145594/workflow)
invalid value workflow reference: references to workflows must be prefixed with format 'owner/repository/' or './' for local workflows